website/client/components/tasks/column.vue

```diff
@@ -69,7 +69,9 @@
   .reward-items {
     display: flex;
     flex-wrap: wrap;
-    justify-content: space-between;
+    > {
```

Comment:
what is this selector for? maybe you meant `& > *` that selects all direct children?

Reply:
My intention (and what just using `>` seems to be doing) was to select all direct children. Since each `shopItem` is constructed with a wrapping `div` I think I'll change this to `& > div` for a more explicit selection of those `divs`. It doesn't seem like the `&` is actually needed here but if that's the convention that should be used I will stick with it.